src/app/components/topnavigation/topnavigation.component.html

```diff
@@ -0,0 +1,9 @@
+<nav class = "top-navigation">
+  <div class="top-navigation__header-block">
+  <div class="top-navigation__name" *scText="rendering.fields.heading"></div>
+</div>
```

Comment:
something seems wonky with <div> closing tag, should it be on line 2?

Reply:
There is header block and heading as title, I need this block for future updates for router(add remove back button for cart view)